packages/onboarding-ui/src/pages/OauthAuthorizationPage/OauthAuthorizationPage.tsx

```diff
@@ -0,0 +1,90 @@
+import { Box, Button } from '@rocket.chat/fuselage';
+import colors from '@rocket.chat/fuselage-tokens/colors.json';
+import type { ReactElement } from 'react';
+import { Trans, useTranslation } from 'react-i18next';
+
+import BackgroundLayer from '../../common/BackgroundLayer';
+import { OnboardingLogo } from '../../common/OnboardingLogo';
+
+type OauthAuthorizationPageProps = {
+  clientName?: string;
+  onClickAuthorizeOAuth: () => void;
+  error: {
+    message?: string;
+    onGoBack?: () => void;
+  };
+};
+
+const OauthAuthorizationPage = ({
+  clientName,
+  onClickAuthorizeOAuth,
+  error,
+}: OauthAuthorizationPageProps): ReactElement => {
+  const { t } = useTranslation();
+
+  const name = clientName || '...loading...';
+
+  return (
+    <BackgroundLayer>
+      <Box
+        display='flex'
+        flexDirection='column'
+        alignItems='center'
+        textAlign='center'
+        width='100%'
+        maxWidth={576}
+        paddingBlock={32}
+        paddingInline={16}
+      >
+        <OnboardingLogo tag='Cloud' />
```

Comment:
We aren't using the tag cloud on logo, is necessary to remove this prop.

Reply:
Ok, tag removed.